tools/smart-domain/smart-domain.xml

```diff
@@ -0,0 +1,104 @@
+<tool id="smart-domain" name="SMART domain" version="0.1.0">
+    <description>search protein domains in sequence using SMART</description>
+    <requirements>
+        <requirement type="package" version="1.6.924">perl-bioperl</requirement>
+        <requirement type="package" version="1.6.924">perl-json</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+<![CDATA[
+perl '$__tool_directory__/smart-domain.pl' 
+--inputFile='$inputFile' 
+$Pfam 
+$SignalP 
+$Repeats 
+$DISEMBL 
+$Schnipsel 
+$outputFormat 
+> $output
+]]>
+    </command>
+    <inputs>
+        <param name="inputFile" type="data" format="fasta" label="FASTA file" help="Single or mutiple protein FASTA sequences" />
+        <param name="Pfam" type="boolean" truevalue="--includePfam" falsevalue="" label="Include Pfam domains in the search" help="--includePfam" />
+        <param name="SignalP" type="boolean" truevalue="--includeSignalP" falsevalue="" label="Include signal peptide predictions" help="--includeSignalP" />
+        <param name="Repeats" type="boolean" truevalue="--includeRepeats" falsevalue="" label="Include internal repeat predictions" help="--includeRepeats" />
+        <param name="DISEMBL" type="boolean" truevalue="--includeDISEMBL" falsevalue="" label="Include predictions of internal protein disorder" help="--includeDISEMBL" />
+        <param name="Schnipsel" type="boolean" truevalue="--includeSchnipsel" falsevalue="" label="Include predictions of outlier homologues and homologues of known structures" help="--includeDISEMBL" />
```

Comment:
Change `help=` to `argument=` for documenting the upstream tool options.

Also the last one should be `--includeSchnipsel` .